react/features/lobby/components/native/LobbyScreen.js

```diff
@@ -75,14 +69,24 @@ class LobbyScreen extends AbstractLobbyScreen<Props> {
 
     _onSwitchToPasswordMode: () => void;
 
-    _onToggleChat: () => void;
```

Comment:
This is no longer helpful because isChatOpen is always false and right now we navigate.